specifyweb/frontend/js_src/lib/components/WbPlanView/mappingPreview.ts

```diff
@@ -26,7 +26,7 @@ import { getMappingLineData } from './navigator';
 
 /** Use table name instead of field name for the following fields: */
 const fieldsToHide = new Set<string>([
-  'name',
+  // 'name',
```

Comment:
Don't comment it out. Rather, remove it entirely.
Don't want to end up like sp6 with a ton of commented-out code that no one knows what do to with